sec-http3/src/config.rs

```diff
@@ -2,31 +2,53 @@ use crate::proto::varint::VarInt;
 
 /// Configures the HTTP/3 connection
 #[derive(Debug, Clone, Copy)]
+#[non_exhaustive]
 pub struct Config {
     /// Just like in HTTP/2, HTTP/3 also uses the concept of "grease"
     /// to prevent potential interoperability issues in the future.
     /// In HTTP/3, the concept of grease is used to ensure that the protocol can evolve
     /// and accommodate future changes without breaking existing implementations.
-    pub send_grease: bool,
+    pub(crate) send_grease: bool,
```

Comment:
My leave the user to configure.

Right now the `Config` struct as a whole is not accessible by name (sealed) so it does not make any difference right now.